src/AutoRest.CSharp/Output/Models/Types/TenantDetection.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License
+using System;
+using System.Collections.Generic;
+using AutoRest.CSharp.Output.Models.Requests;
+using AutoRest.CSharp.Input;
+
+namespace AutoRest.CSharp.Output.Models.Type.Decorate
+{
+    internal static class TenantDetection
```

Comment:
Please move to `Models/Mgmt`

Reply:
this isn't what we will end up with in the end so id rather only move this once  It is scheduled for a future PR https://dev.azure.com/azure-mgmt-ex/DotNET%20Management%20SDK/_workitems/edit/5725